docs/usage/key-concepts/pull-requests.md

```diff
@@ -7,23 +7,31 @@ This page describes how Renovate pull requests work.
 
 ## How Renovate finds existing PRs
 
-Renovate does not keep any kind of database/state of its own about open or closed Pull Requests.
+Renovate does not keep any database/state about open or closed Pull Requests.
```

Comment:
```suggestion
Renovate does not need to maintain any database/state about open or closed Pull Requests.
```

We do kind of keep state with repositoryCache, so I want to loosen the wording a little